src/turbomind/models/llama/LlamaBatch.cc

```diff
@@ -715,7 +715,7 @@ void LlamaBatch<T>::AllocateBuffer(size_t batch_size, size_t session_len, int ca
         context_decoder_output_buf_ = (T*)allocator_->reMalloc(context_decoder_output_buf_, sz, false);
     }
     else {
-        context_decoder_output_buf_ = (T*)allocator_->reMalloc(
+        context_decoder_output_buf_ = (T*)peer_allocator_->reMalloc(
```

Comment:
Now that we have `allocator_` and `peer_allocator_`, which buffers should be allocated with `allocator_` and which should be allocated with `peer_allocator_`? And why?

Reply:
the ones that will be passed to NCCL should be allocated by peer_allocator